README.en.md

```diff
@@ -0,0 +1,192 @@
+# Node-RED (dashboard V2) installation & user guide
+
+*Document version : V1*
+
+[![en](https://img.shields.io/badge/lang-fr-red.svg)](https://github.com/PlanktoScope/node-red-gui-refonte/tree/feat/doc/README.md)
+
+## **Summary**
+
+- *Introduction*
+- *Requirements*
+- *Installation on PlanktoScope*
+
+## **Introduction**
+
+Node-RED is a visual development tool for connecting hardware devices, APIs and online services in new and exciting ways. It offers an intuitive interface for building application flows by connecting nodes that represent distinct blocks of functionality. In this documentation, we'll guide you through the process of installing, configuring and using Node-RED, particularly in the context of PlanktoScope.
+
+## **Prerequisites**
+
+**PlanktoScope**: Make sure PlanktoScope is correctly installed and accessible via SSH.  
+**GitHub account**: A GitHub account is required to manage projects and access tokens.  
+**SSH access**: You must be able to connect to your PlanktoScope via SSH to modify Node-RED configuration files.
+
+
+## **Installation on the PlanktoScope**.
+
+1. **Accessing the PlanktoScope V1 Dashboard** (in French)
+
+After inserting an SD card containing the PlanktoScope image, the V1 dashboard is available online. You can access the PlanktoScope landing page at the following address:
+
+**Landing page :** `http://<planktoscope-ip-address>`
+
+From this page, several options are available, including :
+
+* **Node-RED dashboard editor :** This link gives you access to the dashboard editor. This editor allows you to modify and customize the various elements of the dashboard.
+
+* **Node-RED dashboard:** This link takes you to the V1 version of the dashboard. However, please note that once you have installed version V2, this link will no longer work.
```

Comment:
The existing version of the dashboard is actually v2, so we should call the new version (corresponding to this repo) version v3.

We also need to add the url of the new link to use once version v3 is installed.